### ISSUE TITLE
`oasis serve`

### ISSUE THREAD

**Issue by sharmaster96** [oasislabs/oasis-cli#117]
In the spirit of abstracting away all things blockchain, we may want to consider renaming the `oasis chain` command to `oasis serve`, since the local blockchain behaves quite similarly to just running a local web server.

**Comment by nhynes** [comment]
`oasis serve` sounds like it might also serve your app. Maybe something closer to `oasis devserver` but less verbose. What do you think?

**Comment by sharmaster96** [comment]
Yeah, I agree. Do you have any ideas for a less verbose version of `oasis devserver`? Two other ideas might be `oasis start` or `oasis run`, but especially for `oasis run` it feels too ambiguous given it doesn't take any arguments.
